internal/keycloakb/accountingclient.go

```diff
@@ -0,0 +1,47 @@
+package keycloakb
+
+import (
+	"context"
+
+	cs "github.com/cloudtrust/common-service/v2"
+	"gopkg.in/h2non/gentleman.v2/plugin"
+	"gopkg.in/h2non/gentleman.v2/plugins/headers"
+	"gopkg.in/h2non/gentleman.v2/plugins/url"
+)
+
+const (
+	getBalancePath = "/realms/:realm/users/:userID/balance?service=:service"
+
+	prmUserID    = "userID"
+	prmRealmName = "realm"
+	prmService   = "service"
+	hdrCorrID    = "X-Correlation-ID"
+)
+
+type AccountingClient struct {
+	httpClient HttpClient
+}
+
+type HttpClient interface {
+	Get(data interface{}, plugins ...plugin.Plugin) error
+}
+
+type AccountingBalance struct {
+	Balance *float32 `json:"balance"`
```

Comment:
float64 is used in accounting (but I don't think it can cause an issue with our values)

Reply:
I will change to have the same type here !